packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -748,6 +751,11 @@ export default class NewVm extends BaseComponent {
     template => template && template.virtualizationMode === 'hvm'
   )
 
+  _templateNeedVtpm = createSelector(
+    () => this.props.template,
+    template => template && Boolean(template.platform?.vtpm)
+  )
```

Comment:
I'm not sure this requires a selector. It's very little computation and it's only used once in the `render`. Or was it only for code mutualization?

Reply:
This was mainly for code mutualization.